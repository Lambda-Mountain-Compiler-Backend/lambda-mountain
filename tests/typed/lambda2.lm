
f := Î»(: x X)(: y Y). x;
