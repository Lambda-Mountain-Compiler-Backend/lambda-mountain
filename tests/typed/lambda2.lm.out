(() (Global (f (: ((Lambda ((App ((App ((App ((Literal :) (Variable x))) (Literal X))) (App ((App ((Literal :) (Variable y))) (Literal Y))))) (: ((Variable x) X)))) (-> ((Cons (X Y)) X)))))))
